app/models/status_message.rb

```diff
@@ -149,10 +149,5 @@ def absence_of_content
   def self.tag_stream(tag_ids)
     joins(:taggings).where('taggings.tag_id IN (?)', tag_ids)
   end
-
-  def after_parse
```

Comment:
this is a leftover from the old federation and can be removed. Maybe there is more unused stuff, that I haven't found yet, sorry for that.